diff --git a/react-native-nitro-ota/nitrogen/generated/ios/NitroOta-Swift-Cxx-Bridge.cpp b/react-native-nitro-ota/nitrogen/generated/ios/NitroOta-Swift-Cxx-Bridge.cpp
index c1e95dc..7a6d8c8 100644
--- a/react-native-nitro-ota/nitrogen/generated/ios/NitroOta-Swift-Cxx-Bridge.cpp
+++ b/react-native-nitro-ota/nitrogen/generated/ios/NitroOta-Swift-Cxx-Bridge.cpp
@@ -40,7 +40,12 @@ namespace margelo::nitro::nitroota::bridge::swift {
   // pragma MARK: std::shared_ptr<HybridNitroOtaSpec>
   std::shared_ptr<HybridNitroOtaSpec> create_std__shared_ptr_HybridNitroOtaSpec_(void* NON_NULL swiftUnsafePointer) noexcept {
     NitroOta::HybridNitroOtaSpec_cxx swiftPart = NitroOta::HybridNitroOtaSpec_cxx::fromUnsafe(swiftUnsafePointer);
-    return std::make_shared<margelo::nitro::nitroota::HybridNitroOtaSpecSwift>(swiftPart);
+    if(DEBUG){
+      return std::make_shared<margelo::nitro::nitroota::HybridNitroOtaSpecSwift>(std::move(swiftPart));
+    }
+    else{
+      return std::make_shared<margelo::nitro::nitroota::HybridNitroOtaSpecSwift>(swiftPart);
+    }
   }
   void* NON_NULL get_std__shared_ptr_HybridNitroOtaSpec_(std__shared_ptr_HybridNitroOtaSpec_ cppType) {
     std::shared_ptr<margelo::nitro::nitroota::HybridNitroOtaSpecSwift> swiftWrapper = std::dynamic_pointer_cast<margelo::nitro::nitroota::HybridNitroOtaSpecSwift>(cppType);
diff --git a/react-native-nitro-ota/nitrogen/generated/ios/c++/HybridNitroOtaSpecSwift.hpp b/react-native-nitro-ota/nitrogen/generated/ios/c++/HybridNitroOtaSpecSwift.hpp
index 921a50f..705a20a 100644
--- a/react-native-nitro-ota/nitrogen/generated/ios/c++/HybridNitroOtaSpecSwift.hpp
+++ b/react-native-nitro-ota/nitrogen/generated/ios/c++/HybridNitroOtaSpecSwift.hpp
@@ -35,9 +35,14 @@ namespace margelo::nitro::nitroota {
   class HybridNitroOtaSpecSwift: public virtual HybridNitroOtaSpec {
   public:
     // Constructor from a Swift instance
-    explicit HybridNitroOtaSpecSwift(const NitroOta::HybridNitroOtaSpec_cxx& swiftPart):
-      HybridObject(HybridNitroOtaSpec::TAG),
-      _swiftPart(swiftPart) { }
+    explicit HybridNitroOtaSpecSwift(NitroOta::HybridNitroOtaSpec_cxx swiftPart)
+      : HybridObject(HybridNitroOtaSpec::TAG)
+      #if DEBUG
+            , _swiftPart(std::move(swiftPart))
+      #else
+            , _swiftPart(swiftPart)
+      #endif
+    { }
 
   public:
     // Get the Swift part
